tests/PHPStan/CapitalizationOfIDRuleIntegrationTest.php

```diff
@@ -0,0 +1,95 @@
+<?php declare(strict_types=1);
+
+namespace MLL\Utils\Tests\PHPStan;
+
+use PHPStan\Analyser\Analyser;
+use PHPStan\Analyser\Error;
+use PHPStan\Testing\PHPStanTestCase;
+use PHPUnit\Framework\Attributes\DataProvider;
+
+final class CapitalizationOfIDRuleIntegrationTest extends PHPStanTestCase
+{
+    /** @return iterable<array{0: string, 1: array<int, array<int, string>>}> */
+    public static function dataIntegrationTests(): iterable
+    {
+        self::getContainer();
+
+        yield [__DIR__ . '/data/wrong-capitalization.php', [
+            5 => ['Name of Stmt_Class "LabIdProcessor" should use "ID" instead of "Id", rename it to "LabIDProcessor".'],
+            7 => ['Name of Stmt_ClassMethod "getLabId" should use "ID" instead of "Id", rename it to "getLabID".'],
```

Comment:
```suggestion
            5 => [
                'Name of Stmt_Class "LabIdProcessor" should use "ID" instead of "Id", rename it to "LabIDProcessor".',
            ],
            7 => [
                'Name of Stmt_ClassMethod "getLabId" should use "ID" instead of "Id", rename it to "getLabID".',
            ],
```